src/utils/EventBus/EventBus.ts

```diff
@@ -0,0 +1,64 @@
+import EventEmitter from 'eventemitter3';
```

Comment:
why not to use native `EventTarget`?

Reply:
70+ kb is too much for such a simple thing, I guess.

Reply:
Replaced with EventTarget